File: app/src/main/java/com/example/musicwiki/repo/network/topgenre/Attr.java

package com.example.musicwiki.repo.network.topgenre;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Attr {

    @SerializedName("offset")
    @Expose
    public Integer offset;
    @SerializedName("num_res")
    @Expose
    public Integer numRes;
    @SerializedName("total")
    @Expose
    public Integer total;

}

File: app/src/main/java/com/example/musicwiki/models/Attr.kt
package com.example.musicwiki.models

import com.google.gson.annotations.SerializedName
import com.google.gson.annotations.Expose

data class Attr (
    @SerializedName("page")
    @Expose
    val page: String? = null,

    @SerializedName("perPage")
    @Expose
    val perPage: String? = null,

    @SerializedName("totalPages")
    @Expose
    val totalPages: String? = null,

    @SerializedName("total")
    @Expose
    val total: String? = null
)